charts/lagoon-remote/Chart.yaml

```diff
@@ -19,7 +19,7 @@ type: application
 # This is the chart version. This version number should be incremented each
 # time you make changes to the chart and its templates, including the app
 # version.
-version: 0.96.2
+version: 0.96.3
```

Comment:
No need to bump the version in a PR :wink: 
```
suggestion
version: 0.96.2
```

Reply:
yeah, i had every intention of this being its own release, but the other things got merged before